DeAceroDLabs/marketplace-frontend
# [DMP-661] Added quantity field

## What is done

[[DMP-661] Agregar Campo de Unidades](https://deacero.atlassian.net/browse/DMP-661)

## How is done

1. Added an input for the user to indicate how many units of a product they want.
2. Added an error validation so the input can't be empty nor be lower than 1.

## How to test

1. Run the project via `npm start`.
3. Click on a product from the products list on the home page.
4. You should be sent to the product detail view.
5. You should see the quantity field on the right side of the page.
6. Modify it. It should work without a problem.
7. Try to put a 0 or lower number. You shouldn't be able to do so.
